.junie/plans/add-posthog-mcp.md

```
@@ -0,0 +1,308 @@
+---
+sessionId: session-260809-102650-1r91
+---
+
+# Requirements
+
+### Overview & Goals
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)